### PR TITLE
Add page with list of deployed contracts on mainnet

### PR DESCRIPTION
**Pull request by CJ42**
image

### CODE REVIEW

File: src/components/DeployedContractsTable/contracts-versions.json [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
...f657600080fd5b83018035915067ffffffffffffffff82111561101157600080fd5b602001915036819003821315610c0457600080fd5b8183823760009101908152919050565b60006020828403121561104857600080fd5b6107f182610dd1565b60005b8381101561106c578181015183820152602001611054565b50506000910152565b848682376000858201600081528551611092818360208a01611051565b018385823760009301928352509095945050505050565b60208152600082518060208401526110c8816040850160208701611051565b601f01601f19169190910160400192915050565b600085516110ee818460208a01611051565b855190830190611102818360208a01611051565b0183858237600093019283525090949350505050565b6000825161112a818460208701611051565b9190910192915050565b8a815260006001600160a01b03808c16602084015260e0604084015261115e60e084018b8d610da8565b8915156060850152838103608085015261117981898b610da8565b905081871660a085015283810360c0850152611196818688610da8565b9e9d5050505050505050505050505050565b89815260c0602082015260006111c260c083018a8c610da8565b881515604084015282810360608401526111dd81888a610da8565b90506001600160a01b038616608084015282810360a0840152611201818587610da8565b9c9b50505050505050505050505056fea2646970667358221220add0ea42f8f9e02abd8c7da64d0b13eef395e008fa30377a6b79ea444e7d21bb64736f6c63430008110033",
```
Comment (JeneaVranceanu) [inline src/components/DeployedContractsTable/contracts-versions.json:14]: `creationBytecode` was added but I do not see it being used. Should be accessible in any way?

**Review by JeneaVranceanu — APPROVED** [review]
Didn't we have 0.15.* version deployed as well? 🤔

**Comment by JeneaVranceanu** [comment]
> Didn't we have 0.15.* version deployed as well? 🤔

Looks like they weren't deployed.

File: docs/contracts/deployed-contracts.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: ⛓ Deployed Contracts
```
Comment (Hugoo) [inline docs/contracts/deployed-contracts.md:2]: Can you add metadata too? A description would be nice. You can see the syntax on other pages

https://docusaurus.io/docs/markdown-features/head-metadata

I think page description would be nice.

File: src/components/DeployedContractsTable/index.tsx [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+  const baseContractNames = Object.keys({
+    UniversalProfileInit,
+    LSP6KeyManagerInit,
+    LSP1UniversalReceiverDelegateUP,
+  });
+
+  return baseContracts.map((baseContract, index) => {
+    return baseContract.versions.map(({ version, address, releaseurl }) => (
+      <>
```
Comment (Hugoo) [inline src/components/DeployedContractsTable/index.tsx:28]: This fragment is probably not needed

File: src/components/DeployedContractsTable/index.tsx [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+export default function DeployedContractsTable() {
+  const { address: lsp23Address } = LSP23LinkedContractsFactory;
+  const { address: postDeploymentModuleAddress } =
+    UniversalProfileInitPostDeploymentModule;
+
+  return (
+    <table>
+      <thead>
+        <tr>
+          <td>Contract</td>
```
Comment (Hugoo) [inline src/components/DeployedContractsTable/index.tsx:65]: You need to use th here instead

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/th

**Review by Hugoo — CHANGES_REQUESTED** [review]

**Review by Hugoo — APPROVED** [review]

**Review by skimaharvey — APPROVED** [review]

**Review by YamenMerhi — APPROVED** [review]

File: src/components/DeployedContractsTable/contracts-versions.json (hunk 1 above)
Comment (CJ42) [reply src/components/DeployedContractsTable/contracts-versions.json:14]: I added it for future reference if someone is looking. The plan is to make a separate PR where you can see the bytecode + compiler settings used in the UI.

**Pull request merged**